adoc/headers/multipointerlegacy.h

```diff
@@ -61,7 +62,7 @@ class [[deprecated]] multi_ptr<ElementType, Space, access::decorated::legacy> {
   // Returns the underlying OpenCL C pointer
   pointer_t get() const;
 
-  std::add_pointer_t<value_type> get_raw() const;
+  std::add_pointer_t<element_type> get_raw() const;
```

Comment:
Why change this now that you've added the `value_type` alias?  It seems better to keep it as `std::add_pointer_t<value_type>` to be consistent with the non-legacy specialization.

Reply:
Good catch, this was a typo, should have been `value_type`, thanks. Same for the second case too.